pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -41,11 +41,16 @@ def __init__(self, trainer: "pl.Trainer", resume_from_checkpoint: Optional[_PATH
         self._loaded_checkpoint: Dict[str, Any] = {}
 
     @property
-    def hpc_resume_path(self) -> Optional[str]:
+    def auto_restart_path(self) -> Optional[str]:
```

Comment:
don't we need to rename this in a different PR?

Reply:
Reverted back.